webrtc.html

```diff
@@ -1751,20 +1768,18 @@ <h4>Set the RTCSessionDescription</h4>
                             description</a>.</p>
                           </li>
                           <li>
-                            <p>If no suitable transceiver is found
+                            <p>If a suitable transceiver was found (<var>transceiver</var>
+                            is set) and <var>sendEncodings</var> is non-empty, set
+                            <var>transceiver</var>.<a>[[\Sender]]</a>.<a>[[\sendEncodings]]</a>
+                            to <var>sendEncodings</var>, and set
+                            <var>transceiver</var>.<a>[[\Sender]]</a>.<a>[[\LastReturnedParameters]]</a>
+                            to <code>null</code>.</p>
```

Comment:
Does this mean we allow modifying sendEncodings in subsequent offers for transceivers that already exist?

Reply:
No, suitable = unused. See https://github.com/w3c/webrtc-pc/issues/2147#issuecomment-477355180.

Reply:
Ack!